autoarray/inversion/regularization/constant.py

```diff
@@ -46,11 +46,10 @@ def regularization_matrix_from(self, mapper):
 
 
 class ConstantSplit(Constant):

```

Comment:
Splitting up the regularization classes is mainly for autofit, which cannot make model parameters from the if_interpolated bool.

Still, I would argue this is a nicer syntax, as we dont have to write `if self.if_interpolated` within methods and simply overwrite them based on class inheritance.